README.md

```diff
@@ -1880,16 +1876,17 @@ Any good resources on this?
 
 ## Taxes on Equity Compensation
 
-Now that we’ve covered the basic concepts of equity and taxes, we’ll get into some messy
-details of how they interact.
-
-As already discussed, employees can get restricted stock, stock options, or RSUs.
-The tax consequences for each of these is dramatically different.
+It is crucial to understand the tax implications of stock options and awards.
```

Comment:
This changes sense. The first sentence now neglects RSUs.

I'll take another pass at rewriting these.

Reply:
I just cut the first sentence, it was padding.